### DISCUSSION TITLE
File converter error

### DISCUSSION THREAD

**Discussion by ghost** [Ryder25/Ego-Engine-Modding#64] [Q&A] [answered]
Hello, I'm trying to convert a pkg of f1 2011 to a ego engine file, but when I use the file converter .exe always appear to be an error:
Failed to convert the file!
System.AggregateException: Could not determine the file type! Showing json, and xml errors:  ---> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: ⌂. Path '', line 0, position 0.

Please help me, I'm trying anything about 2 weeks.

**Comment by ptasev** [comment, marked as answer]
Which file is it? The pkg files that are supported are from F1 2015+ so the one from F1 2011 may be a completely different file format.

**Reply by ghost** [reply]
It's pkg, previously it was a nfs file format, I tried to find a way to open but couldn't find, so I converted to a pkg file an then in the ego file converter but error, maybe the problem it's what you said that only works from f12015+

**Reply by ptasev** [reply]
If it is an nfs file you can try opening it with NeFS Edit:
https://github.com/victorbush/ego.nefsedit

**Reply by ghost** [reply]
Very thanks, but how do I open the file? I'm not finding any exe or cmd

**Reply by ptasev** [reply]
You can find the exe in a zip file on the releases page:
https://github.com/victorbush/ego.nefsedit/releases

**Reply by ghost** [reply]
Sadly it only opens nefs and I have nfs, I'm gonna try to maybe convert to nefs but that will be hard to do.

Regardless Thanks a lot :)
